src/ui/actions/AppActions.ts

```diff
@@ -109,6 +109,10 @@ const init = async () => {
     await PlayerActions.next();
   });
 
+  ipcRenderer.on('playback:stop', async () => {
```

Comment:
It doesn't need to be `async` if it does not return a `Promise` or if it does not use the `await` keyword.